src/operations/channels.jl

```diff
@@ -0,0 +1,323 @@
+"""
+    SplitChannels <: Augmentor.Operation
+
+Description
+--------------
+
+Splits out the color channels of the given image using the
+function `ImageCore.channelview`. This will effectively create a
+new array dimension for the colors in the front. In contrast to
+`ImageCore.channelview` it will also result in a new dimension
+for Gray images.
```

Comment:
Interesting, I didn't know there were frameworks that did this. You may have noticed that ImageCore at one time supported a flag that allowed one to toggle between these behaviors, but I suspect support for that has decayed since it hasn't been used.